source/manual/test-and-build-a-project-on-jenkins-ci.html.md

```diff
@@ -63,6 +63,31 @@ Many GOV.UK applications test against the
 To test your application for each PR on govuk-content-schemas, add it to the [govuk-content-schemas
 Jenkinsfile](https://github.com/alphagov/govuk-content-schemas/blob/master/Jenkinsfile).
 
+## Using Docker databases
+
+We have utilised Docker as a way to provide access to different versions of
+databases which can be used as part of your build. These can be accessed with
+the following connection strings:
```

Comment:
I'd be tempted to reframe this section to be tech-agnostic – for example, "Specify which database version to use" or similar.

```suggestion
## Specifying which database version to use

There are different versions of the MySQL and Postgres database engines which
can be used as part of your build. These can be accessed with
the following connection strings:
```

The fact that the different database versions run using docker seems like an implementation detail that probably isn't relevant here.